pipes/rules/hs_deplete.rules

```diff
@@ -55,18 +55,17 @@ rule filter_to_taxon:
     '''
     input:  
         input_bam = config["data_dir"]+'/'+config["subdirs"]["depletion"]+'/{sample}.cleaned.bam',
-        lastalDb  = objectify_remote(expand('{lastal_ref_db_dir}/lastal.{ext}', lastal_ref_db_dir=config["lastal_ref_db_dir"], ext=['bck','des','prj','sds','ssp','suf','tis'])),
+        lastalDb  = objectify_remote(expand('{lastal_ref_db_dir}/lastal.fasta', lastal_ref_db_dir=config["lastal_ref_db_dir"])),
```

Comment:
@tomkinsc can you quickly take a peek and see if these changes I made to pipes/rules/hs_deplete.rules look okay? I also deleted the `build_lastal_db` rule from ncbi.rules (but not the `download_lastal_sources`)--the snakemake rules should be expecting only a `lastal.fasta` file as input/output. `taxon_filter.py filter_lastal_bam` accepts a naked fasta file as input for its db now (and that functionality is now unit tested).

Reply:
They look ok! The input variable could be renamed from `lastalDb` to `lastal_db_fasta` or similar to reflect that it is not actually the database. Do we still want to wrap the fasta file with `objectify_remote()` if the assumption is that the DB will usually be built quickly on the fly? Can't hurt to keep remote support I guess.

It could make to change [`lastal_ref_db_dir`](https://github.com/broadinstitute/viral-ngs/blob/master/pipes/config.yaml#L258) in `config.yml` to `lastal_db_fasta`,  and point to the source fasta rather than a directory containing the DB, with the assumption that the DB files would be alongside the fasta in the same directory.

Reply:
hm.. that's a good point on `objectify_remote`. how about this, since it looks functional, I'll go ahead and merge this in so we can have a functioning `master` branch again, and then I'll let you improve that rule at a later time after we get a few other necessary bits in.

Reply:
Sounds good!